tests/main.test.ts

```diff
@@ -37,7 +37,7 @@ describe("Plugin tests", () => {
 
   it("Should serve the manifest file", async () => {
     const worker = (await import("../src/worker")).default;
-    const response = await worker.fetch(new Request("http://localhost/manifest.json"), {
+    const response = await worker.fetch(new Request("http://localhost/manifest"), {
```

Comment:
I don't like this removal of the file name extension.

Reply:
It's very strange to hit endpoints with extensions except to download files usually.